libraries/freertos_plus/standard/freertos_plus_tcp/source/FreeRTOS_TCP_IP.c

```diff
@@ -2671,6 +2671,8 @@ BaseType_t xSendLength = 0;
 uint32_t ulReceiveLength;	/* Number of bytes contained in the TCP message. */
 uint8_t *pucRecvData;
 uint32_t ulSequenceNumber = FreeRTOS_ntohl (pxTCPHeader->ulSequenceNumber);
+configASSERT(pxSocket);
```

Comment:
Would it make sense to move this assert higher, i.e. make it a precondition

Reply:
Moved it.